ansible/roles/xroad-ss/tasks/rhel.yml

```diff
@@ -21,6 +53,14 @@
 - name: add xroad admin user
   command: "xroad-add-admin-user {{ xroad_ui_user }}"
 
+- name: disable service postgresql and ensure it is masked in case we are using remote database server
+  systemd:
+    name: postgresql
+    state: stopped
+    enabled: no
+    masked: yes
+  when: database_admin_password != ""
```

Comment:
* could be lifted to the main.yml to avoid duplication (same for RHEL and Ubuntu)
* stopping the db can break things:
  * the configured "remote database" used could actually be the local host db
  * there can be other uses for the local db --> stopping it should at least be configurable